src/main/java/com/uber/jenkins/phabricator/CommentBuilder.java

```diff
@@ -82,6 +83,7 @@ public void processParentCoverage(CodeCoverageMetrics parentCoverage, String bas
         String lineCoverageDisplay = String.format("%.3f", lineCoveragePercent);
 
         if (coverageDelta > 0) {
+            passCoverage = false;
```

Comment:
Checking simply for whether the delta is greater or less than 0 is a big limitation. Maybe we should make it so that we fail if coverage drops below X%, where X is user defined and has a clear minimum if we enable this check? Thoughts?

/cc @sectioneight, who I've had this same conversation about a long time ago :)